mungegithub/mungers/approval-handler.go

```diff
@@ -351,6 +356,11 @@ func createApproverSet(comments []*githubapi.IssueComment) sets.String {
 			if cmd.Name != approveCommand {
 				continue
 			}
+
```

Comment:
Did you add these extra spaces so that I wouldn't complain about something else? didn't work :-/